examples/cc3220sf/xively_demo/.project

```diff
@@ -28,7 +29,7 @@
 	<variableList>
 		<variable>
 			<name>XIVELY_LIBRARY_C_ROOT</name>
-			<value>file:/E:/dev/xively/xively-client-c</value>
+			<value>$%7BPARENT-3-PROJECT_LOC%7D</value>
```

Comment:
This is from the test I ran on the last 3220 PR. `$%7BPARENT-3-PROJECT_LOC%7D` is eclipse-talk for `../../../`

Reply:
Eclipse ... you're a confusing IDE.